ci-operator/config/openshift/cluster-api-provider-baremetal/openshift-cluster-api-provider-baremetal-master.yaml

```diff
@@ -49,7 +49,7 @@ tests:
   steps:
     cluster_profile: equinix-ocp-metal
     workflow: baremetalds-e2e-ovn-ipv6
-- as: e2e-metal-ipi-upgrade
+- as: e2e-metal-ipi-upgrade-ovn-ipv6
```

Comment:
Network type still is openshift sdn?

Reply:
since it's v6, it will get converted to ovnk in devscripts, iirc